Project.toml

```diff
@@ -31,13 +31,22 @@ StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 UUIDs = "cf7118a7-6976-5b1a-9a39-7adc72f591a4"
 
 [compat]
-FixedPointNumbers = "≥ 0.3.0"
-GR = "≥ 0.31.0"
-PlotThemes = "≥ 0.1.3"
-PlotUtils = "≥ 0.6.0"
-RecipesBase = "≥ 0.6.0"
-StatsBase = "≥ 0.14.0"
-julia = "≥ 1.0.0"
+Contour = "0.5"
+FFMPEG = "0.2"
+FixedPointNumbers = "0.3, 0.6"
+GR = "0.31, 0.44"
+GeometryTypes = "0.7"
+JSON = "0.21"
+Measures = "0.3"
+NaNMath = "0.3"
+PlotThemes = "0.2, 1"
+PlotUtils = "0.6.1"
+RecipesBase = "0.6, 0.7"
+Reexport = "0.2"
+Requires = "0.5"
+Showoff = "0.3.1"
+StatsBase = "0.14, 0.32"
```

Comment:
This seems wrong, now e.g. 0.15 will not be supported.

Reply:
Since this PR is about PGFPlotsX I would just leave the compat as is maybe?

Reply:
That was from an earlier PR from myself, and just got here after the rebase.

Reply:
Oh